src/containers/WelcomePage/components/worklist/StatusCell.tsx

```diff
@@ -24,8 +24,8 @@ const TextWrapper = styled.div`
 `;
 
 const StyledCheckIcon = styled(Check)`
-  height: ${spacing.snormal};
-  width: ${spacing.snormal};
+  height: 20px;
```

Comment:
Kommentererer her for å separerere litt fra den eksisterende diskusjonen om ikon-størrelser. Hvorfor ikke disse bare være enten 24px eller 16px?

Reply:
Ikoner som har hover/fokus innhold, eller attpåtil er knapper, må minimum ha størrelse, eller størrelse+padding, som tilsammen utgjør 24px ref [2.5.8 i WCAG 2.2](https://tetralogical.com/blog/2023/10/05/whats-new-wcag-2.2/#2.5.8-target-size-(minimum)-(aa)). Det utelukket 16px. Når jeg la dette inn, prøvde jeg med 24px, men syntes ikonene ble hakket for store og dominerende for tabellene, derfor brukte jeg 20px som størrelse, slik det også var gjort i tegningen under How to meet the requirement.
Om jeg er eneste som synes ikonene ble for store på 24px, så setter jeg meg ikke på bakbeina :)

Reply:
Om jeg ikke husker feil har vi et minstekrav for høyde i `IconButton`, så ikonstørrelse skal strengt tatt ikke ha allverdens å si når det kommer til UU. Utseendemessig er jo en helt annen greie igjen. Kan fort se for meg at 16 og 24 kan være for stort/ for lite.

Reply:
Mange av designtegningene til Hedvig bruker 20px på ikoner, men det er gjerne inni knapper og slikt.